src/Persistent/Api.hs

```diff
@@ -325,7 +366,7 @@ instance FromProto IdeaLike where
 -- | FIXME: Same user can like the same idea more than once (Issue #308).
 -- FIXME: Assumption: the given @AUID Idea@ MUST be in the DB.
 addLikeToIdea :: AUID Idea -> AddDb m IdeaLike
-addLikeToIdea iid = addDb (dbIdeaMap . at iid . _Just . ideaLikes)
+addLikeToIdea iid = undefined -- TODO: addDb (dbIdeaMap . at iid . _Just . ideaLikes)
```

Comment:
Here you need to give a name to `dbIdeaMap . at iid . _Just . ideaLikes`, this makes one constructor `DbIdeaLike :: AUID Idea -> DbField IdeaLikes` and the code goes to `dbFieldSetter`.